.github/workflows/ci.yml

```diff
@@ -22,6 +22,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install -r requirements.txt
+        cp .env.example .env
```

Comment:
We should be using GITHUB environment variables during the CI/CD rather than creating a .env file

Reply:
If you don't have access to set up the environment variables in GH for actions let me know